CHANGELOG.rst

```diff
@@ -7,13 +7,25 @@ The format is largely inspired by keepachangelog_.
 
 .. _0.1.1:
 
+v0.6.3 - Unreleased
+===================
+
+Bugfixes
+--------
+
+- Disallow registering schemas with typecodes that conflict
+  with builtin types' typecodes. This would cause all sorts
+  of subtle breakage.
+
 v0.6.2 - 2019-02-21
 ===================
 
 Bugfixes
 --------
 
-- Map only requested size, when specified, in id mappers.
+- Disallow registering schemas with typecodes that conflict
```

Comment:
Why did you remove the original message?

Reply:
Bad merge